modules/cluster-logging-deploy-cli.adoc

```diff
@@ -113,95 +95,55 @@ spec: {}
 ----
 <1> You must specify the `openshift-operators-redhat` namespace.
 
-.. Create an Operator Group object:
-+
-[source,terminal]
-----
-$ oc create -f <file-name>.yaml
-----
-+
-For example:
+.. Apply the `OperatorGroup` object by running the following command:
 +
 [source,terminal]
 ----
-$ oc create -f eo-og.yaml
+$ oc apply -f <filename>
 ----
 
-.. Create a Subscription object YAML file (for example, `eo-sub.yaml`) to
-subscribe a namespace to the OpenShift Elasticsearch Operator.
+.. Create a `Subscription` object to subscribe a namespace to the {es-op}:
 +
-.Example Subscription
+.Example `Subscription` object
 [source,yaml]
 ----
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: "elasticsearch-operator"
-  namespace: "openshift-operators-redhat" <1>
+  name: elasticsearch-operator
+  namespace: openshift-operators-redhat <1>
 spec:
-  channel: "stable-5.5" <2>
-  installPlanApproval: "Automatic" <3>
-  source: "redhat-operators" <4>
-  sourceNamespace: "openshift-marketplace"
-  name: "elasticsearch-operator"
+  channel: <channel> <2>
+  installPlanApproval: Automatic <3>
+  source: redhat-operators <4>
+  sourceNamespace: openshift-marketplace
+  name: elasticsearch-operator
 ----
 <1> You must specify the `openshift-operators-redhat` namespace.
-<2> Specify `stable`, or `stable-5.<x>` as the channel. See the following note.
+<2> Specify `stable`, or `stable-y.z` as the channel, where `y` is the major version and `z` is the minor version. See the following note.
 <3> `Automatic` allows the Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available. `Manual` requires a user with appropriate credentials to approve the Operator update.
 <4> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster,
 specify the name of the CatalogSource object created when you configured the Operator Lifecycle Manager (OLM).
 +
 [NOTE]
 ====
-Specifying `stable` installs the current version of the latest stable release. Using `stable` with `installPlanApproval: "Automatic"`, will automatically upgrade your operators to the latest stable major and minor release.
+Specifying `stable` installs the current version of the latest stable release. Using `stable` with `installPlanApproval: "Automatic"` automatically upgrades your Operators to the latest stable major and minor release.
 
-Specifying `stable-5.<x>` installs the current minor version of a specific major release. Using `stable-5.<x>` with `installPlanApproval: "Automatic"`, will automatically upgrade your operators to the latest stable minor release within the major release you specify with `x`.
+Specifying `stable-y.z` installs the current minor version of a specific major release. Using `stable-y.z` with `installPlanApproval: "Automatic"` automatically upgrades your Operators to the latest stable minor release within the major `y` release.
```

Comment:
Do you mean the minor or patch/z release? We usually refer to minor as y, and z as "patch" or just "z" releases. It's major-minor-patch: 4.14.3, for example.

Reply:
For logging we only refer to major and minor, per https://docs.openshift.com/container-platform/4.14/logging/cluster-logging-upgrading.html
Do you know if the major-minor-patch thing is documented anywhere? It might be helpful to point to as a follow up.